docs/userguide/monitoring.rst

```diff
@@ -0,0 +1,55 @@
+Monitoring
+==========
+
+Parsl aims to make the task of running parallel workflows easy by providing monitoring and diagnostic
+capabilities to help track the state of your workflow down to the individual applications being
+executed on remote machines. To enable Parsl's monitoring feature for your workflow you will need
+a few additional packages.
+
+Installation
+------------
+
+Parsl's monitoring model relies on writing workflow progress to a sqlite database and separate tools
+that query this database to create web-based dashboard for the workflow.
+
+To enable workflow monitoring support install::
+
+    $ pip install parsl[monitoring]
+
+Monitoring configuration
+------------------------
+
+Here's an example configuration that logs monitoring information to a local sqlite database.
+
+.. code-block:: python
+
+    config = Config(
+        executors=[
+            HighThroughputExecutor(
+                label="local_htex",
+                cores_per_worker=1,
+                address=address_by_hostname(),
+            )
+        ],
+        monitoring=MonitoringHub(
+            hub_address=address_by_hostname(),
+            hub_port=55055,
+            logging_level=logging.INFO,
+            resource_monitoring_interval=10,
+        ),
+        strategy=None
+    )
+
+
+
+Visualization
+-------------
+
+Install the visualization server::
+
+   $ pip install git+https://github.com/Parsl/viz_server.git
```

Comment:
if this is coming from a separate repo (which is probably the wrong thing to do), this line should have some commentary on how to get the version that works with whichever version of parsl you've got installed.

Reply:
The `viz_server` is not ready for merging back yet, but we want it merged back by 0.8.0. So this is only a means for getting an alpha for getting user feedback on viz. I expect that the github repo will only work for Parsl >= 0.7.2, <= 0.8.0.